src/averageAges.js

```diff
@@ -14,12 +14,29 @@
  *
  * @return {number}
  */
+
+function filterSex(array, sex) {
+  return array.filter(person => person.sex === sex);
+};
+
+function calcAvarage(number, divider) {
+  return Number((number / divider).toFixed(2));
+}
```

Comment:
in a good way, this function should receive an array of digits, sum them up and divide the sum by the length of this array